styles.css

```diff
@@ -9,6 +9,204 @@ body {
   background-repeat: no-repeat;
 }
 
+/* start of css modal */
+
+.modal-mobile {
+  width: 100%;
+  height: 836px;
+  border-radius: 8px;
+  cursor: pointer;
+  box-shadow: 0 0 50px rgba(255, 255, 255, 0.3);
+  transition-property: box-shadow;
+  transition-duration: 0.6s;
+  transition-timing-function: linear;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  overflow-y: auto;
+  overflow-x: hidden;
+  z-index: 999;
+}
+
+.moblie-model {
+  position: fixed;
+  z-index: 999;
+  background: #fff;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  overflow-y: auto;
+  padding: 16px;
+  align-items: flex-start;
+}
+
+.mainimage {
+  display: flex;
+  margin: 0 auto;
+  width: 100%;
+}
+
+.closeimage {
+  display: flex;
+  align-self: flex-end;
+  justify-content: flex-end;
+  z-index: 999;
+  position: absolute;
+  right: 38px;
+  height: 20px;
+  cursor: pointer;
+}
```

Comment:
- [ ] Please use the same close button as in Figma desing for mobile version notice that there is different close button for the mobile version and desktop version

| Your design | Figma design |
| ------------- | --------------  |
| ![image](https://user-images.githubusercontent.com/74744787/162448422-2d46a681-5679-441e-a469-72e8c2779af3.png) | ![image](https://user-images.githubusercontent.com/74744787/162448642-c6bed89b-e4b4-428e-8839-5f7214678263.png) |
|